cobrand_hackney/static/js.js

```diff
@@ -37,3 +37,69 @@ document.querySelectorAll('.case-detail__map__overlay').forEach(function(b) {
 });
 
 })();
+
+
+/* Creating a new filter called "Case Locations"  */ 
+// Fixes issue https://github.com/mysociety/noiseworks/issues/53
```

Comment:
Don’t think we need these two lines. You’ve already included a link to the issue in the commit message :+1: